Python/pystate.c

```diff
@@ -652,6 +652,14 @@ PyInterpreterState_New(void)
 
     init_interpreter(interp, runtime, id, old_head, pending_lock);
 
+#if WITH_FREELISTS
+    for (int i=0; i < INTERP_NUM_FREELISTS; i++) {
+        _PyFreeList_Init(&interp->freelists[i],
+                         FREELIST_INDEX_TO_SIZE(i),
+                         SMALL_OBJECT_FREELIST_SIZE);
+    }
+#endif
```

Comment:
I guess we could just remove the WITH_FREELIST guard everywhere and just initialise here with capacity 0 if it's flipped on?